### DISCUSSION TITLE
Режим планирования

### DISCUSSION THREAD

**Discussion by GorD876** [voxeltycoon/issues#1008] [General]
Можно добавить режим планирования построек. В этом режиме вы можете экспериментировать с постройкой масштабных сооружений и оценивать их стоимость без траты бюджета на ошибки. После завершения строительства в этом режима, можно будет построить всё за один клик.

**Comment by MajestyCorp** [comment]
Если на паузе строить и удалять здания - возвращается полная стоимость :)
